contracts/validator-manager/ACP99ValidatorManager.sol

```diff
@@ -0,0 +1,60 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+import {IACP99SecurityModule} from "./interfaces/IACP99SecurityModule.sol";
+import {ValidatorManager} from "./ValidatorManager.sol";
+import {IACP99ValidatorManager, ConversionData, ValidatorRegistrationInput} from "./interfaces/IACP99ValidatorManager.sol";
+
+pragma solidity 0.8.25;
+
+abstract contract ACP99ValidatorManager is IACP99ValidatorManager, ValidatorManager {
+    IACP99SecurityModule public securityModule;
+
+    // TODO: calling this should be restricted to...who?
+    function setSecurityModule(IACP99SecurityModule _securityModule) external {
+        securityModule = _securityModule;
+    }
```

Comment:
Should the security module itself control its own upgrades? If not, should there be a `SecurityUpdateModule`?